pandas/tests/io/formats/test_to_excel.py

```diff
@@ -1,6 +1,6 @@
 """Tests formatting as writer-agnostic ExcelCells
 
-ExcelFormatter is tested implicitly in pandas/tests/io/test_excel.py
+ExcelFormatter is tested implicitly in pandas/tests/io/excel tests
```

Comment:
Could also change this to the actual file pandas/tests/io/excel/test_style.py

Reply:
Had the same idea initially, but we are running through this class from all files except test_odf and test_readers. I though referencing the folder would be better in this case.

Should I change it?

Reply:
Up to you, either works (this also wouldn't have to change again if / when tests get moved around)

Reply:
Yes, had similar thoughts. I would keep it like it is now then. Thx

Reply:
Very nit-picky, but you might even drop the last word "tests" as redundant